src/main/scala/org/scalameter/Persistor.scala

```diff
@@ -6,15 +6,15 @@ package org.scalameter
 
 
 trait Persistor {
-  def load(context: Context): History
-  def save(context: Context, h: History): Unit
+  def load[T: Manifest](context: Context): History[T]
+  def save[T](context: Context, h: History[T]): Unit
```

Comment:
I haven't made `Persistor` generic since that would be more painful to the users when they would like to change default `Measurer` (need of overriding `def persistor: Persistor[T]` too).
The drawback of it is that only Jackson needs `Manifest` to deserialization, but in the above approach all the reporters that deals with persisting data require `Manifest` to call `persistor.load` even if `persistor` is in fact `SerializationPersistor`.